src/pages/_app.js

```diff
@@ -25,9 +25,8 @@ function MyApp({ Component, pageProps }) {
       <Head>
         <meta charSet="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <link href="https://fonts.googleapis.com/css?family=Be+Vietnam&display=swap" rel="stylesheet" />
         <link
-          href="https://fonts.googleapis.com/css2?family=Be+Vietnam:wght@500&display=swap"
+          href="https://fonts.googleapis.com/css2?family=Be+Vietnam:wght@300;400;500;600&display=swap"
```

Comment:
Removed old duplicate font embeds for "Be Vietnam" and edited the embed to include a couple more weights. Might be worth noting that Google Fonts only lists "Be Vietnam Pro" on their site - just thought I'd share in case we want to transition to the "pro" version in the future.